scimath/StatsFramework/StatisticsDataset.h

```diff
@@ -255,32 +254,32 @@ class StatisticsDataset {
     );
     // </group>
 
-    // instead of setting and adding data "by hand", set the data provider that will provide
-    // all the data sets. Calling this method will clear any other data sets that have
-    // previously been set or added.
+    // instead of setting and adding data "by hand", set the data provider that
+    // will provide all the data sets. Calling this method will clear any other
+    // data sets that have previously been set or added.
     void setDataProvider(StatsDataProvider<CASA_STATP> *dataProvider);
 
 private:
-    std::vector<DataIterator> _data;
-    // maps data to weights.
-    // maps are used rather than vectors because only some (or none) of the data
-    // sets in the _data vector may have associated weights, masks, and/or ranges.
-    std::map<uInt, WeightsIterator> _weights;
+    std::vector<DataIterator> _data{};
+    // maps data to weights. maps are used rather than vectors because only some
+    // (or none) of the data sets in the _data vector may have associated
+    // weights, masks, and/or ranges.
+    std::map<uInt, WeightsIterator> _weights{};
     // maps data to masks
-    std::map<uInt, MaskIterator> _masks;
-    std::vector<Int64> _counts;
-    std::vector<uInt> _dataStrides;
-    std::map<uInt, uInt> _maskStrides;
-    std::map<uInt, Bool> _isIncludeRanges;
-    std::map<uInt, DataRanges> _dataRanges;
-    StatsDataProvider<CASA_STATP> *_dataProvider;
-
-    Int64 _idataset;
-    typename std::vector<DataIterator>::const_iterator _dend, _diter;
-    std::vector<Int64>::const_iterator _citer;
-    std::vector<uInt>::const_iterator _dsiter;
-    uInt _dataCount;
-    ChunkData _chunk;
+    std::map<uInt, MaskIterator> _masks{};
+    std::vector<Int64> _counts{};
+    std::vector<uInt> _dataStrides{};
+    std::map<uInt, uInt> _maskStrides{};
+    std::map<uInt, Bool> _isIncludeRanges{};
+    std::map<uInt, DataRanges> _dataRanges{};
+    StatsDataProvider<CASA_STATP>* _dataProvider{nullptr};
+
+    Int64 _idataset{0};
+    typename std::vector<DataIterator>::const_iterator _dend{}, _diter{};
+    std::vector<Int64>::const_iterator _citer{};
+    std::vector<uInt>::const_iterator _dsiter{};
+    uInt _dataCount{0};
+    ChunkData _chunk{};
```

Comment:
This change breaks the clang Travis build, see https://travis-ci.org/casacore/casacore/jobs/414161122#L8291